skimage/measure/_find_contours_cy.pyx

```diff
@@ -13,14 +16,17 @@ cdef inline double _get_fraction(double from_value, double to_value,
 
 
 def iterate_and_store(double[:, :] array,
-                      double level, Py_ssize_t vertex_connect_high):
+                      double level, Py_ssize_t vertex_connect_high,
+                      cnp.uint8_t[:, :] mask):
```

Comment:
I'm surprised that we don't get a `ValueError: Does not understand character buffer dtype format string ('?')` here when passing in a mask of dtype('bool') like you mentioned previously in https://github.com/scikit-image/scikit-image/pull/4150#discussion_r324168938. I still get this error when using the `cnp.uint8_t[::1]` notation in another PR. Perhaps that's why...

See also https://github.com/cython/cython/pull/2676#issuecomment-529916887.